src/arrayMethodSort.js

```diff
@@ -4,8 +4,28 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction) {
+    for (let i = 0; i < this.length; i++) {
+      for (let j = i; j < this.length; j++) {
```

Comment:
The inner loop should start from `i + 1` instead of `i` to avoid unnecessary comparisons and swaps when `i` is equal to `j`.